src/api/bookstore.js

```diff
@@ -0,0 +1,50 @@
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+
+export const initialize = async () => {
+  if (!localStorage.getItem('appId')) {
+    const response = await fetch(`${BASE_URL}/apps/`, {
+      method: 'POST',
+    });
+    const appId = await response.text();
+    localStorage.setItem('appId', appId);
+  }
+};
```

Comment:
- [ ] And also the fetch method for the API is not rendering anything  👉 [see the screenshot here](https://user-images.githubusercontent.com/61562626/130690319-17f1d8f1-9abc-42e5-bfa4-35f1da143441.png).

- [ ] Please use the `componentDidMount` lifecycle method or with react hooks requests. see the screenshot 👇

![nbnbn](https://user-images.githubusercontent.com/61562626/130689595-0b216f5d-b196-4d80-be16-39d4e7e8a03a.jpeg)

Reply:
Well done to use hooks.